src/lib/events/click.js

```diff
@@ -53,25 +60,25 @@ document.addEventListener('click', bubbly.evaluateEvent, true);
 
 /**
  * The click event. This event occurs when a user has clicked an element.
- * @param {Object} settings The event settings object.
- * @param {string} [settings.elementSelector] The CSS selector the element must match in order for
+ * @param settings - The event settings object.
+ * @param {string} settings.elementSelector - The CSS selector the element must match in order for
```

Comment:
You may be aware of this already, but the square brackets is notation in JSDoc for indicating that the property is optional, which I believe is still the case for these properties.